docs/user-quick-start.md

```diff
@@ -0,0 +1,299 @@
+   [...]
+   2023-11-02 05:40:40,777 INFO    [rdr] Dumping ramen e2e config to '/home/aclewett/.config/drenv/rdr'
+   2023-11-02 05:40:41,129 INFO    [rdr] Environment started in 839.40 seconds
+   ```
+The kubconfigs for dr1, dr2 and hub can be found in $HOME/.config/drenv/rdrkubeconfigs/.
+
+
+Validate that the clusters are created.
+
+   ```
+   minikube profile list
+   ```
+   
+   Example output:   
+   ```
+   |---------|-----------|------------|-----------------|------|---------|---------|-------|--------|
+   | Profile | VM Driver |  Runtime   |       IP        | Port | Version | Status  | Nodes | Active |
+   |---------|-----------|------------|-----------------|------|---------|---------|-------|--------|
+   | dr1     | kvm2      | containerd | 192.168.122.70  | 8443 | v1.27.4 | Running |     1 |        |
+   | dr2     | kvm2      | containerd | 192.168.122.51  | 8443 | v1.27.4 | Running |     1 |        |
+   | hub     | kvm2      | containerd | 192.168.122.162 | 8443 | v1.27.4 | Running |     1 |        |
+   |---------|-----------|------------|-----------------|------|---------|---------|-------|--------|
+   ```
+   
+You can inspect the environment using these commands.
+
+   ```
+   kubectl get deploy -A --context hub
+   kubectl get deploy -A --context dr1
+   kubectl get deploy -A --context dr2
+   ```
+
+## Build the ramen operator image
+
+Build the *Ramen* operator container image:
+
+  ```
+  cd ramen
+  make docker-build
+  ```
+
+This builds the image `quay.io/ramendr/ramen-operator:latest`
+
+## Deploy and Configure the ramen operator
+
+To deploy the *Ramen* operator in the test environment:
+
+  ```
+  ramenctl deploy test/envs/regional-dr.yaml
+  ```
+
+For more info on the `ramenctl` tool see
+[ramenctl/README.md](../ramenctl/README.md).
+
+Ramen now needs to be configured to know about the managed clusters and the
+s3 endpoints. To configure the ramen operator for test environment:
+
+   ```
+   ramenctl config test/envs/regional-dr.yaml
+   ```
+
+## Running system tests
+
+At this point *Ramen* is ready to protect workloads in your cluster, and
+you are ready for testing basic flows.
+
+To run basic tests using regional-dr environment run:
+
+   ```
+   cd ramen
+   test/basic-test/run test/envs/regional-dr.yaml
+   ```
+
+This test:
+
+1. Deploys an application using
+   [ocm-ramen-samples repo](https://github.com/RamenDR/ocm-ramen-samples)
+1. Enables DR for the application
+1. Fails over the application to the other cluster
+1. Relocates the application back to the original cluster
+1. Disables DR for the application
+1. Uninstalls the application
+
+If needed, you can run one or more steps from this test, for example to
+deploy and enable DR run:
+
+   ```
+   env=$PWD/test/envs/regional-dr.yaml
+   test/basic-test/deploy $env
+   test/basic-test/enable-dr $env
+   ```
+
+At this point you can run run manually failover, relocate one or more
+times as needed.
+
+Video demo: https://youtu.be/8-fpChSWzeo
```

Comment:
Awseome!

Maybe add the link at the top of the page - under the title?

Reply:
I added it here because it goes with running ramen basic-tests explained in this section

Reply:
ok, it would be nice to use a markdown link with an image preview:

[![Video demo](https://img.youtube.com/vi/8-fpChSWzeo/hqdefault.jpg)](https://www.youtube.com/embed/8-fpChSWzeo)